blog/models.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+
+    def __str__(self):
+        return self.username
+
+    class Meta:
+        verbose_name = "user"
+        verbose_name_plural = "users"
+        db_table = "users"
+        ordering = ["username"]
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="posts"
+    )
+    title = models.CharField(max_length=63)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
+
+    def commentary_count(self):
+        return self.commentary.count()
```

Comment:
The method name 'commentary_count' suggests it returns a count, but it actually returns a queryset. Consider renaming it to something like 'get_commentary_count' or modifying it to return the count using 'self.commentary.count()'.